pkg/reconciler/namespace/namespace.go

```diff
@@ -174,6 +190,14 @@ func (r *Reconciler) reconcileServiceAccountAndRoleBindings(ctx context.Context,
 		return fmt.Errorf("track role binding '%s': %v", rb.Name, err)
 	}
 
+	if sa.Name == resources.IngressServiceAccountName || sa.Name == resources.FilterServiceAccountName {
+		_, err := CopySecret(r, "default", env.BrokerPullSecretName, ns.Name, sa.Name)
```

Comment:
The source secret will probably be in `knative-eventing`. Use `system.Namespace()`

Reply:
done, thanks.